pkg/cloudprovider/provider/aws/provider.go

```diff
@@ -111,6 +111,18 @@ var (
 				owner: "125523088429",
 			},
 		},
+		providerconfigtypes.OperatingSystemRockyLinux: {
+			awstypes.CPUArchitectureX86_64: {
+				description: "Rocky Linux Base System",
+				// The AWS marketplace ID from Rocky Linux Community Platform Engineering (CPE)
+				owner: "057448758665",
```

Comment:
Where's this owner ID coming from? I can't find it on AWS, I can only find the other one (`679593333241`).

Reply:
Yeah, for Kubeone terraform I've found `792107900819`. Found by inspecting AMIs from here https://rockylinux.org/cloud-images.

Reply:
```shell
        {
            "Architecture": "x86_64",
            "CreationDate": "2021-08-04T08:03:36.000Z",
            "ImageId": "ami-042fac9db82d0b975",
            "ImageLocation": "057448758665/Rocky Linux 8.4 - HVM",
            "ImageType": "machine",
            "Public": true,
            "OwnerId": "057448758665",
            "PlatformDetails": "Linux/UNIX",
            "UsageOperation": "RunInstances",
            "State": "available",
            "BlockDeviceMappings": [
                {
                    "DeviceName": "/dev/sda1",
                    "Ebs": {
                        "DeleteOnTermination": true,
                        "SnapshotId": "snap-0e74d57094e56c7f8",
                        "VolumeSize": 10,
                        "VolumeType": "gp2",
                        "Encrypted": false
                    }
                }
            ],
            "Description": "Rocky Linux Base System",
            "Hypervisor": "xen",
            "Name": "Rocky Linux 8.4 - HVM",
            "RootDeviceName": "/dev/sda1",
            "RootDeviceType": "ebs",
            "VirtualizationType": "hvm"
        }
```

Reply:
```shell
$ aws ec2 describe-images --image-ids ami-079f4c25253b27e9d --region eu-central-1 | jq '.Images[] .OwnerId'
"792107900819"

$ aws ec2 describe-images --image-ids ami-0a10fe11f3e1bf41e --region eu-west-1 | jq '.Images[] .OwnerId'
"792107900819"

$ aws ec2 describe-images --image-ids ami-0abb90b1685f9e9fc --region eu-west-3 | jq '.Images[] .OwnerId'
"792107900819"
```

Reply:
The AMI IDs are from the https://rockylinux.org/cloud-images/

Reply:
I mean, 057448758665 is also a existing owner, it's just not the official one, some 3rd party whatever.